catalyst-gateway/bin/src/db/index/session.rs

```diff
@@ -259,14 +260,31 @@ async fn retry_init(cfg: cassandra_db::EnvVars, persistent: bool) {
         };
 
         // Set up the Schema for it.
-        if let Err(error) = create_schema(&mut session.clone(), &cfg).await {
-            let error = format!("{error:?}");
-            error!(
-                db_type = db_type,
-                error = error,
-                "Failed to Create Cassandra DB Schema"
-            );
-            continue;
+        if !created_scheme {
+            if let Err(error) = create_schema(&mut session.clone(), &cfg).await {
+                let error = format!("{error:?}");
+                error!(
+                    db_type = db_type,
+                    error = error,
+                    "Failed to Create Cassandra DB Schema"
+                );
+                continue;
+            }
+            created_scheme = true;
+        }
+
+        let key_space = if persistent {
+            format!("persistent_{}", SCHEMA_VERSION.replace('-', "_"))
```

Comment:
This breaks autogeneration of the keyspace schema version.  The comment was about NOT using it anywhere but in tests.

Autogeneration of the keyspaces schema version is critical to allow us to safely update the schema and not collide with a pre-existing database.